File: sdk-java/src/main/java/com/crosscert/passkey/sdk/exception/PasskeyAuthException.java
package com.crosscert.passkey.sdk.exception;

import com.crosscert.passkey.sdk.envelope.EnvelopeFieldError;

import java.util.List;

public class PasskeyAuthException extends PasskeyApiException {
    public PasskeyAuthException(String code, String message, String traceId,
                                List<EnvelopeFieldError> fieldErrors) {
        super(401, code, message, traceId, fieldErrors);
    }
}

File: sdk-java/src/main/kotlin/com/crosscert/passkey/sdk/exception/PasskeyAuthException.kt
package com.crosscert.passkey.sdk.exception

import com.crosscert.passkey.sdk.envelope.EnvelopeFieldError

class PasskeyAuthException(
    code: String?,
    message: String?,
    traceId: String?,
    fieldErrors: List<EnvelopeFieldError>?,
) : PasskeyApiException(401, code, message, traceId, fieldErrors)
